targets/targets.json

```diff
@@ -1111,9 +1111,8 @@
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM"],
         "program_cycle_s": 2,
         "extra_labels": ["STM", "STM32F4", "STM32F407", "STM32F407xG", "STM32F407VG"],
-        "macros": ["LSI_VALUE=32000"],
+        "macros": ["LSI_VALUE=32000", "TRANSACTION_QUEUE_SIZE_SPI=2"],
```

Comment:
This is something that my linter picked up. While it's not strictly forbidden in JSON, behavior for having multiple keys of the same name is not well defined. I believe this was an oversight by a previous commiter, so I decided to "fix" it by merging the entries. Anyone reading this out of context should look a few lines below in the source, where the duplicated 'macros' entry lives.

Although if merging is not correct in this case, I am willing to change it back to its original state or any of suggested forms.